plugins/inventory/digitalocean.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c), Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+name: digitalocean
+plugin_type: inventory
+author:
+  - Janos Gerzson (@grzs)
+short_description: DigitalOcean Inventory Plugin
+version_added: "0.04"
+description:
+  - DigitalOcean Inventory plugin.
+  - Acquires droplet list from DO api.
+  - Uses configuration file that ends with
+  - (do_hosts|digitalocean|digital_ocean).(yaml|yml)
```

Comment:
```suggestion
  - Uses configuration file that ends with C((do_hosts|digitalocean|digital_ocean).(yaml|yml)).
```
Generally: make sure you have complete sentences per paragraph (each list item is one paragraph), and use the proper formatting (https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation).